website/docs/docs/dbt-versions/release-notes/03-Oct-2023/custom-branch-fix-rn.md

```diff
@@ -0,0 +1,12 @@
+---
+title: "Fix: Default behavior for CI job runs without a custom branch"
+description: "October 2023: CI job runs now default to the main branch of the Git repository when a custom branch isn't set"
+sidebar_label: "Fix: Default behavior for CI job runs without a custom branch"
+tags: [Oct-2023]
+date: 2023-10-06
+sidebar_position: 09
+---
+
+If you don't set a [custom branch](/docs/dbt-cloud-environments#custom-branch-behavior) for your dbt Cloud environment, it now defaults to the `main` branch of your Git repository. Previously, [CI jobs](/docs/deploy/ci-jobs) would run for pull requests (PRs) that were opened against _any branch_ if the **Custom Branch** option wasn't set. 
```

Comment:
`would run for pull requests (PRs) that were opened against _any branch`

@knartovich : should i update this ^ to also include "updated with new commits" ?

Reply:
sure! i think it doesn't hurt to be explicit about that here